andela/mervels-bn-frontend
# #168131536 Profile Information from Travel Request Page(UI)

#### What does this PR do?
Implements auto-filling profile information on the travel request form
#### Description of Task to be completed?
- Add a checkbox that auto-fills the personal info on the request form
- Allow the checkbox to change the default value of the autofill preference if changed by the time it's submitted
#### How should this be manually tested?
- Clone this repo and checkout to this branch `ft-profile-information-on-requests-168131536`
- Run the app `npm start`
- And navigate to `/requests` and start a new request
- You should see an auto-filling option
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#168131536](https://www.pivotaltracker.com/story/show/168131536)
#### Screenshots (if appropriate)
<img width="1206" alt="Screen Shot 2019-11-22 at 14 10 49" src="https://user-images.githubusercontent.com/50402526/69425106-f5b49580-0d32-11ea-92e6-3b6fb8a3c96b.png">

#### Questions:
N/A
